modules/chapter3/pages/section1.adoc

```diff
@@ -1,5 +1,7 @@
 = Introduction
 
+video::jupyter-intro.mkv[width=100%]
```

Comment:
Excellent Jaime! Can you embed the video at the end of this section at the bottom. Add a line saying something like "If you cannot run this demo yourself, please see the demo video of basics of Jupyter notebooks etc"

Reply:
Ahhh that makes sense. I will make the change.